### ISSUE TITLE
Error when importing libcombine from python

### ISSUE THREAD

**Issue by yannikschaelte** [sbmlteam/libCombine#26]
Upon ``import libcombine`` from python, I get the below error. Seems to happen on python3.8, not on 3.6 (used a clean anaconda installation). Further, ``pip install python-libcombine``, gives me version 0.2.1, though 0.2.3 should be the latest? 0.2.3_1 is available too.

```
>>> import libcombine
Traceback (most recent call last):
  File "/home/yannik/anaconda38/envs/env/lib/python3.8/site-packages/libcombine/__init__.py", line 25, in swig_import_helper
    return importlib.import_module(mname)
  File "/home/yannik/anaconda38/envs/env/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 657, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 556, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 1101, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: /home/yannik/anaconda38/envs/env/lib/python3.8/site-packages/libcombine/_libcombine.cpython-38-x86_64-linux-gnu.so: undefined symbol: deflateInit2_

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/yannik/anaconda38/envs/env/lib/python3.8/site-packages/libcombine/__init__.py", line 28, in <module>
    _libcombine = swig_import_helper()
  File "/home/yannik/anaconda38/envs/env/lib/python3.8/site-packages/libcombine/__init__.py", line 27, in swig_import_helper
    return importlib.import_module('_libcombine')
  File "/home/yannik/anaconda38/envs/env/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: /home/yannik/anaconda38/envs/env/lib/python3.8/site-packages/libcombine/_libcombine.cpython-38-x86_64-linux-gnu.so: undefined symbol: deflateInit2_
```

**Comment by yannikschaelte** [comment]
error is indeed gone when I install ``pip install python-libcombine==0.2.3_1`` (giving 0.2.3)

**Comment by fbergmann** [comment]
i'll re-release a newer version, that should take care of the issue. Sorry for the trouble

**Comment by yannikschaelte** [comment]
no problem, works for us. thanks for the fast reply!

**Comment by jonrkarr** [comment]
I've seen the same issue. It appears to be related to Python 3.8.

**Comment by fbergmann** [comment]
this should now been resolved for version 0.2.6.

**Comment by yannikschaelte** [comment]
thanks, it seems resolved!
